### ISSUE TITLE
 NVDA is not announcing the information of coordinates tooltips of the map using Tab/arrow keys.

### ISSUE THREAD

**Issue by yangster23** [WiredSolutions/react-azure-maps#86]
NVDA is not announcing the information of coordinates tooltips of the map using Tab/arrow keys - https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html. 

This issue is also observed with the "Chromium Edge+ Narrator".

NVDA should announce the information of coordinates tooltips of the map using Tab/arrow keys. As a result, screen reader users will not be able to understand the information of coordinates tooltips of the map

**Comment by rbrundritt** [comment]
I believe this is an implementation issue, not a bug in this SDK. The accessible popup example you reference creates a popup for each location. You will need to do the same in your app rather than reusing a single popup instance.
